test/writer.asciidoc

```diff
@@ -466,7 +466,11 @@ Ellipses…and…and….
 * latexmath:[$223$]
 * latexmath:[$p$]-Tree
 * Here’s some display math:
-latexmath:[\[\frac{d}{dx}f(x)=\lim_{h\to 0}\frac{f(x+h)-f(x)}{h}\]]
+
```

Comment:
I'm curious about this example.  Does it actually work properly when formatted using asciidoc?  Do you really get one big list with the math in one list item?